docs/en/ingest-management/elastic-agent/scaling-on-kubernetes.asciidoc

```diff
@@ -0,0 +1,220 @@
+[[scaling-on-kubernetes]]
+= Scaling {agent} on kubernetes
+
+- Please follow the steps to run the {agent} on <<running-on-kubernetes-managed-by-fleet>> page.
+- Please follow the steps to run the {agent} on <<running-on-kubernetes-standalone>> page.
+
+
+
+[discrete]
+== Observability under scale
+
+Using https://www.elastic.co/guide/en/welcome-to-elastic/current/getting-started-kubernetes.html[Elastic Observability] to monitor Kubernetes infrastructure at scale, imposes a higher complexity in order to configure and adjust Elastic Stack based on different kubernetes sizes. The Horizontal Pod scaling changes the shape of your Kubernetes workload by automatically increasing or decreasing the number of Pods, thus this will impact the following:
```

Comment:
Why do we care about Pod autoscaler here? 
I wouldn't expect to read about this at all. The key point here is that when we have large k8s cluster (Nodes+Pods) we face the following.

Reply:
Ok updated. Check latest commit and let me know if u agree